docs/DeveloperGuide.md

```diff
@@ -1,34 +1,528 @@
-# Developer Guide
+# Developer Guide for PlanNUS
 
-## Design & implementation
+<table><tr><td><div style="text-align:center">
+    <img src="images/PlanNUSLogo.png" />
+</div></td></tr></table>
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+Before reading this document, you are recommended to read through the [user guide](https://ay2021s1-cs2113t-f12-1.github.io/tp/UserGuide.html).
 
+## Table of contents 
```

Comment:
Would it be better if u numbered the individual sections for the table of contents? And the sections themselves